app/models/post.rb

```diff
@@ -183,6 +183,10 @@ def self.public?
     self.where(private: false, unlisted: false, draft: false)
   end
 
+  def public?
```

Comment:
Won't this interfere with the definition above? Or are they separate because one is a query and the other is for individual posts?

https://github.com/EloHellEsports/workshop.codes/blob/552091aca2207c4645724ef8e84b3b3bcb02ff42/app/models/post.rb#L182-L184

Reply:
They are separate indeed. `self.public` refers to the class, and is a method to filter a query. `public?` is a method on an instance of the class, and refers only to a particular instance.